static/build-static

```diff
@@ -180,35 +183,62 @@ esac
 # cleanup
 [ -d "${buildDir}" ] && rm -r "${buildDir}"
 
-# docker
-mkdir -p "${dockerBuildDir}"
+# docker CLI
+mkdir -p "${dockerCLIBuildDir}"
 case ${TARGETOS} in
 	linux | darwin)
-		cp "${CLI_DIR}"/build/"${targetPair}"/docker-"${TARGETOS}"-* "${dockerBuildDir}/docker"
+		cp "${CLI_DIR}"/build/"${targetPair}"/docker-"${TARGETOS}"-* "${dockerCLIBuildDir}/docker"
 		;;
 	windows)
-		cp "${CLI_DIR}"/build/"${targetPair}"/docker-"${TARGETOS}"-*.exe "${dockerBuildDir}/docker.exe"
+		cp "${CLI_DIR}"/build/"${targetPair}"/docker-"${TARGETOS}"-*.exe "${dockerCLIBuildDir}/docker.exe"
 		;;
 esac
+# package docker CLI
+case ${TARGETOS} in
+	linux | darwin)
+		(
+			set -x
+			tar -C "${buildDir}" -c -z -f "${buildDir}/docker-cli-${STATIC_VERSION}.tar.tgz" docker-cli
+		)
+		;;
+	windows)
+		(
+			cd "${buildDir}"
+			set -x
+			zip -r "docker-cli-${STATIC_VERSION}.zip" docker-cli
+		)
+		;;
+esac
+
+# docker, containerd, and runc
+mkdir -p "${dockerBuildDir}"
 case ${TARGETOS} in
 	linux)
-		for f in dockerd containerd ctr containerd-shim containerd-shim-runc-v2 docker-init docker-proxy runc; do
+		for f in dockerd docker-init docker-proxy; do
 			if [ -f "${ENGINE_DIR}/bundles/${TARGETPLATFORM}/$f" ]; then
 				cp -L "${ENGINE_DIR}/bundles/${TARGETPLATFORM}/$f" "${dockerBuildDir}/$f"
 			fi
 		done
+		# TODO containerd binaries should be built as part of containerd-packaging, not as part of docker/docker-ce-packaging
+		mkdir -p "${containerdBuildDir}"
+		for f in containerd ctr containerd-shim containerd-shim-runc-v2 runc; do
+			if [ -f "${ENGINE_DIR}/bundles/${TARGETPLATFORM}/$f" ]; then
+				cp -L "${ENGINE_DIR}/bundles/${TARGETPLATFORM}/$f" "${containerdBuildDir}/$f"
+			fi
+		done
 		;;
 	windows)
 		cp "${ENGINE_DIR}"/bundles/"${TARGETPLATFORM}"/dockerd-*.exe "${dockerBuildDir}/dockerd.exe"
 		cp "${ENGINE_DIR}"/bundles/"${TARGETPLATFORM}"/docker-proxy-*.exe "${dockerBuildDir}/docker-proxy.exe"
 		;;
 esac
-# package docker
+# package docker, containerd, and runc
 case ${TARGETOS} in
 	linux | darwin)
 		(
 			set -x
 			tar -C "${buildDir}" -c -z -f "${buildDir}/docker-${STATIC_VERSION}.tar.tgz" docker
```

Comment:
@tianon was suggesting perhaps to rename this one to `docker-engine-<version>`, which is worth considering (also because it's no longer the same as the "old" `docker-<version>` which contained everything

Reply:
yes makes more sense